README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Coverage Status](https://coveralls.io/repos/github/andela/technites-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/technites-bn-backend?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/andela/technites-bn-backend/badge.svg?branch=bg-fix-coveralls-integration-168311192)](https://coveralls.io/github/andela/technites-bn-backend?branch=bg-fix-coveralls-integration-168311192)
```

Comment:
I'm wondering if it is okay to receive the coverage from that branch, instead of develop branch, maybe you can explain to me why you chose it that way, otherwise it looks good to me